src/cosmos/EventFeed.tsx

```diff
@@ -66,15 +70,32 @@ export default function EventFeed({ events, onClear }: EventFeedProps) {
           <Type.Subhead1>clear</Type.Subhead1>
         </button>
       </Row>
+      <Row>
+        <Type.Subhead2>Filter: </Type.Subhead2>
+        <select
+          onChange={(e) => {
+            console.log(e.target.value)
+            onSelectEventType(e.target.value)
+          }}
+          value={selectedEventType}
+        >
+          <option>{SHOW_ALL_EVENTS}</option>
+          {HANDLERS.map((name) => (
+            <option key={name}>{name}</option>
```

Comment:
any chance the handlers have the same name? could also use i (index) as key value for consistency with what you do on line 91

Reply:
leaving this as a runtime check - left a comment stating as such